mta-v6.x/specs/notifications.json

```diff
@@ -12,9 +12,7 @@
       }
     ],
     "servers": [
-      {
-        "url": "http://localhost:7007/api/notifications"
-      }
+     
```

Comment:
I'd prefer aligning the mtav6 and mtav7 with mta and use the notifications in yaml format, preferably, using the extension to include it from the same reference.

Reply:
having said that, I'm going to override this part entirely with the PR to switch to the new notifications API.

Reply:
I guess I'll have to take care for it during the rebase after merging this PR.

Reply:
(in other words, pls ignore those comments :-) )